impl/command_line/arguments.cxx

```diff
@@ -388,12 +472,12 @@ namespace substrate::commandLine
 	}
 
 	// Implementation of the innards of arguments_t as otherwise we get compile errors
-	// NOLINTNEXTLINE(modernize-use-equals-default)
-	arguments_t::arguments_t() noexcept : _arguments{} { }
-	arguments_t::arguments_t(const arguments_t &arguments) noexcept : _arguments{arguments._arguments} { }
-	arguments_t::arguments_t(arguments_t &&arguments) noexcept : _arguments{std::move(arguments._arguments)} { }
-	// NOLINTNEXTLINE(modernize-use-equals-default)
-	arguments_t::~arguments_t() noexcept { }
+	arguments_t::arguments_t() noexcept = default;
+	arguments_t::arguments_t(const arguments_t &arguments) noexcept = default;
```

Comment:
GCC 9 (and not later? strange) complains with the following: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=86583

See https://www.open-std.org/jtc1/sc22/wg21/docs/cwg_active.html#1854 for more. I don't think this is correct, but it may have to be handled.

Reply:
Ah! GCC 9 is correct - std::multiset can throw when copied (allocation) so arguments_t must also throw when copied as _arguments is a std::multiset

Reply:
I take from https://en.cppreference.com/w/cpp/container/multiset/multiset that it can't crash with default empty construction? So I'll wait for GCC 9 to be back to happiness before marking as fixed.

Reply:
Aye, default construction shouldn't allocate, as with most of the STL containers, so that should be safe - it's just copy construction that can.